Casper.Network.SDK.Test/NctlContractTest.cs

```diff
@@ -31,7 +31,9 @@ public async Task DeployContractTest()
                 wasmBytes,
                 _faucetKey.PublicKey,
                 50_000_000_000,
-                _chainName);
+                _chainName,
+                1, //gasPrice=1
```

Comment:
Are we sure this is necessary? There is no dynamic gas pricing on Casper network, so if we are good without having this explicitly, then it might be better to leave it out.

Reply:
Both gasPrice and ttl are optional arguments that default to `1` and `1800000` (or 30m), respectively.
Here, I wanted to use a non-default value for the ttl parameter so that the sdk converts it to the string `1day`. And to do that I needed to specify the gasPrice as well.